x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/MlTasks.java

```diff
@@ -477,4 +478,14 @@ public static String prettyPrintTaskName(String taskName) {
             default -> throw new IllegalArgumentException("unexpected task type [" + taskName + "]");
         };
     }
+
+    /**
+     * Builds the task description from the model id that initiated the task.
+     *
+     * @param modelId a string that identifies the model
+     * @return a string representing the task description
+     */
+    public static String createTaskDescription(String modelId) {
```

Comment:
nit: I think `downloadModelTaskDescription` would fit better with the other naming choices in this class